cpp/daal/src/algorithms/dtrees/forest/classification/df_classification_predict_dense_default_batch_impl.i

```diff
@@ -128,7 +126,11 @@ public:
           _cachedData(nullptr),
           _cachedModel(nullptr),
           _averageTreeSize(0),
-          _cachedNClasses(0)
+          _cachedNClasses(0),
+          _blockSize(0),
```

Comment:
Should the default values be the same as it was previously defined?
#define _DEFAULT_BLOCK_SIZE                      32
#define _DEFAULT_BLOCK_SIZE_COMMON               22
#define _MIN_TREES_FOR_THREADING                 100

Reply:
I've modified; but those values are anyway overridden by a 'setHyperparameters()' call here:
https://github.com/oneapi-src/oneDAL/pull/2692/files/#diff-f7c3689a30067e4f500431476e1897d6949edfa783c36c86a86805d4267d6cd3R314
So, everything works Ok even if no hyperparameters are provided from oneDAL side at all.